.travis.yml

```diff
@@ -156,83 +154,6 @@ jobs:
         - kubectl create -f deploy/namespace.yaml
         - molecule test -s test-local
 
-          #- ${TRAVIS_BUILD_DIR}/scripts/ci/fold_ouput.sh cluster.info.2 "cluster information"  kubectl cluster-info dump
+          #- ${TRAVIS_BUILD_DIR}/tools/scripts/ci/fold_ouput.sh cluster.info.2 "cluster information"  kubectl cluster-info dump
 notifications:
   email: false
-# =======
-os: linux
-dist: xenial
-services: docker
-
-language: go
-go:
-  - "1.13"
-
-env:
-  global:
-    - REPO_OWNER=$(dirname $TRAVIS_REPO_SLUG)
-    - REPO_NAME=$(basename $TRAVIS_REPO_SLUG)
-   
```

Comment:
@dunnevan Just  FYI,  I removed the driver piece, you probably should put it back in when you get to testing it, sorry!